plugins/misp/plugin.spec.yaml

```diff
@@ -564,16 +595,19 @@ actions:
       event:
         title: Event ID
         description: Event ID to append to
+        example: "1"
```

Comment:
Since this is a string, can you please add a better example. Lets say UUID or something like event1 instead of numbers to avoid confusion

Reply:
Sure, i just use bigger number